### PR TITLE
Add webworkify for markdown-it processor

### PR DESCRIPTION
**Pull request by tayjiehao**

### CODE REVIEW

File: sashimi-webapp/src/logic/documentPackager/index.js [hunk 1]
```diff
@@ -1,31 +1,13 @@
...
-const processMarkdown = (markdownString =>
-  new Promise((resolve, reject) => {
-    try {
-      resolve(markdownProcessor.process(markdownString));
-    } catch (error) {
-      reject(error);
-    }
-  })
-);
+const w = work(require.resolve('./worker.js'));
```
Comment (chuajiaxuan) [inline sashimi-webapp/src/logic/documentPackager/index.js:4]: Should the "require.resolve('./worker.js')" be changed to another import? E.g. "import worker from './worker.js'"???

**Review by chuajiaxuan — COMMENTED** [review]
See Comment! Otherwise good to go!

File: sashimi-webapp/src/logic/documentPackager/index.js (hunk 1 above)
Comment (tayjiehao) [reply sashimi-webapp/src/logic/documentPackager/index.js:4]: This is part of the `webworkify` required parameter.

**Comment by amoshydra** [comment]
@tayjiehao @chuajiaxuan 

`array.includes` no longer work inside our phantomjs.
However, Jia Xuan have been using `array.includes` without any problem previously.
The exact same code [passed the test](https://travis-ci.org/nus-mtp/sashimi-note/jobs/219227135) in travis + phantomjs 9 days ago

I guess we will have to use the old for loop for the task then.
See the changes > 1f959e8

**Review by amoshydra — APPROVED** [review]

**Review by jiayingy — APPROVED** [review]

**Pull request merged**